app.py

```diff
@@ -0,0 +1,8 @@
+
+
+if __name__ == "__main__":
+    from services import DataBase
+    from url_parser import AuthorMetadataParser
+    db = DataBase("./data/Spotify_Youtube.csv")
+    author = "Muse"
+    print(list(db.filter([lambda x: x.artist == author, lambda x: "hole" in x.track.lower()], mode=all)))
```

Comment:
it looks cool, but i guess that it would be hard to understand how it works

Reply:
I'll spend a little time explaining how it works.